target/linux/ramips/dts/RAK633.dts

```diff
@@ -0,0 +1,97 @@
+/dts-v1/;

```

Comment:
@eikepedia  Please can you consider adding `SPDX-License-Identifier: GPL-2.0-or-later OR MIT` as in [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist) ?
> Having MIT as alternative is sometimes preferred by upstream maintainers
and allows sharing that simple code with other projects. We don't really
want multiple DTS versions for the same device.